ru/lessons/specifics/eex.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Embedded Elixir (EEx)
+category: specifics
+order: 3
+lang: ru
+---
+
+По аналогии с ERB в Ruby и JSP в Java, в Elixir присутствует EEx или Embedded Elixir ("встраиваемый" Elixir код). С помощью EEx мы можем встраивать и интерпретировать код Elixir внутри любого другого текста.
+
+{% include toc.html %}
+
+## API
+
+API EEx позволяет напрямую работать как со строками, так и с файлами.  API разделено на три основных компонента: простая интерпретация, определение функций и компиляция в AST (Абстрактное синтаксическое дерево).
+
+### Интерпретация
+
+Используя функции [`eval_string`](http://elixir-lang.org/docs/stable/eex/EEx.html#eval_string/3) и [`eval_file`](http://elixir-lang.org/docs/stable/eex/EEx.html#eval_file/3) мы можем выполнить интерпретацию кода, содержащегося в строке или файле. Это самые простые функции в API но и самые медленные, т.к. код каждый раз интерпретируется снова а не компилируется заранее.
+
+```elixir
+iex> EEx.eval_string "Hi, <%= name %>", [name: "Sean"]
+"Hi, Sean"
+```
+
+### Определение функций
+
+Более производительный и предпочтительный способ использования EEx, это создание функции из нашего шаблона с кодом, определив её внутри модуля который будет скомпилирован. Для этого мы должны подготовить шаблон до начала компиляции и использовать макросы [`function_from_string`](http://elixir-lang.org/docs/stable/eex/EEx.html#function_from_string/5) или [`function_from_file`](http://elixir-lang.org/docs/stable/eex/EEx.html#function_from_file/5).
```

Comment:
I think that a `&mdash;` should be used instead of a comma between `EEx` and `это` in the first sentence.
Also a comma is missing between `модуля` и `который`.